langchain/callbacks/streaming_aiter_final_only.py

```diff
@@ -83,6 +83,6 @@ async def on_llm_new_token(self, token: str, **kwargs: Any) -> None:
                     self.queue.put_nowait(t)
             return
 
-        # If yes, then put tokens from now on
-        if self.answer_reached:
+        # If answer is not yet reached, then put the token in the queue
+        if not self.answer_reached:
```

Comment:
i think the idea is we only start streaming once we get to the final answer

Reply:
Thanks for replying! I suppose that it could be that my agent never outputs the final answer tokens. I'll look  into this a bit more and update this